lib/rules/newline-after-var.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @fileoverview Rule to check empty newline after "var" statement
+ * @author Gopal Venkatesan <http://g13n.me>
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+    var mode = context.options[0],
+        validator;              // regex to validate the rule
+
+    if (mode === "always") {
+        validator = /\S(?:\r?\n){2,}\S?/;
+    } else if (mode === "never") {
+        validator = /\S(?:\r?\n)?\S/;
```

Comment:
I have addressed this by disabling the option in case of invalid value since I don't want to assume anything about the project.

I may be wrong with my assumption.